database/repository/audit/postgres/audit.go

```diff
@@ -0,0 +1,73 @@
+package audit
+
+import (
+	"fmt"
+
+	"github.com/thrasher-corp/gocryptotrader/database"
+	"github.com/thrasher-corp/gocryptotrader/database/models"
+	"github.com/thrasher-corp/gocryptotrader/database/repository/audit"
+	log "github.com/thrasher-corp/gocryptotrader/logger"
+)
+
+type auditRepo struct{}
+
+// Audit returns a new instance of auditRepo
+func Audit() audit.Repository {
+	return &auditRepo{}
+}
+
+// AddEvent writes event to database
+// writes are done using a transaction with a rollback on error
+func (pg *auditRepo) AddEvent(event *models.AuditEvent) {
+	if pg == nil {
+		return
+	}
+	query := `INSERT INTO audit_event (type, identifier, message) VALUES($1, $2, $3)`
+	tx, err := database.Conn.SQL.Begin()
+	if err != nil {
+		return
+	}
+	_, err = tx.Exec(query, &event.Type, &event.Identifier, &event.Message)
+	if err != nil {
+		_ = tx.Rollback()
```

Comment:
It would be nice to know if Exec failed and what err it had, like what you've done in AddEventTx